integration-cli/docker_cli_daemon_test.go

```diff
@@ -2679,84 +2678,6 @@ func (s *DockerDaemonSuite) TestDaemonRestartSaveContainerExitCode(c *check.C) {
 	c.Assert(out, checker.Equals, errMsg1)
 }
 
-func (s *DockerDaemonSuite) TestDaemonBackcompatPre17Volumes(c *check.C) {
```

Comment:
This change should probably go into the other commit that removes the old code

Reply:
Fixed